starter-code/lib/viking.js

```diff
@@ -19,13 +19,83 @@
 //   FootViking: FootViking
 // };
 
-function Soldier () {}
+function Soldier () {
+  this.health   =300;
+  this.strength =150;
+  this.attack=function(){
+  };
+  this.receiveDamage=function(damage){
+//    var damage=50;
+    this.health-=damage;
+  };
+}
 
-function Viking () {}
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
 
-function Saxon () {}
+function Viking (name, health, strength) {
+  this.name     =name;
```

Comment:
here, you can use `Soldier.call(this, health, strength)` to skip lines 37-39.  You can review the lecture on prototypal inheritance to see an example where we did it with `animal`